ecs-web/task-definitions/create_concourse_db_container.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name"   : "create_db",
+  "image"  : "${image}:${image_tag}",
+  "memoryReservation": 100,
+  "command": [
+    "/bin/sh",
+    "-exc",
```

Comment:
I'd like to see a test before creation but it's just a concern

Reply:
In another situation I would agree, but in this case I think it's not worth it.

It's a really simple command: _try to create a database and user_. If the database doesn't exist it'll create it and move on with the concourse web container. If it already exists, it'll exit with code 1 and will move on with the concourse web container. In both cases we get the expected outcome, so I don't see the point of over-complicating the command with checks that will bring no added value.